<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSystemSkyBox</class>
 <widget class="QDialog" name="DialogSystemSkyBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelFront">
         <property name="text">
          <string>Front:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="WidgetPicture" name="widgetPictureFront" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="6">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>118</width>
           <height>188</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelBack">
         <property name="text">
          <string>Back:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="WidgetPicture" name="widgetPictureBack" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelTop">
         <property name="text">
          <string>Top:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="WidgetPicture" name="widgetPictureTop" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelBottom">
         <property name="text">
          <string>Bottom:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="WidgetPicture" name="widgetPictureBottom" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelLeft">
         <property name="text">
          <string>Left:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="WidgetPicture" name="widgetPictureLeft" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelRight">
         <property name="text">
          <string>Right:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="WidgetPicture" name="widgetPictureRight" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetPicture</class>
   <extends>QWidget</extends>
   <header>widgetpicture.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSystemSkyBox</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSystemSkyBox</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
